### ISSUE TITLE
AutoCompleteTextView and Robolectric Limitations

### ISSUE THREAD

**Issue by joshallenit** [robolectric/robolectric#6912]
### Description

Hi, this is more of a question as to what the systemic limitations of Robolectric are, rather than asking for a fix.

I am trying to test clicking on a drop down selection of an AutoCompleteTextView. I created a test in [/androidTest](https://github.com/joshallenit/issue-robolectric-autocompletetextview/blob/main/app/src/androidTest/java/com/example/reproduce_robo_autocomplete/AutoCompleteTextViewAndroidTest.kt) and a [near identical test under /test](https://github.com/joshallenit/issue-robolectric-autocompletetextview/blob/main/app/src/test/java/com/example/reproduce_robo_autocomplete/AutoCompleteTextViewTest.kt). FragmentScenario + Espresso is used to do the actual testing.

Is whether Robolectric is intended to work in this case? I would like to know if the "write-once, run-anywhere" idea is ever going to be viable.

Thanks!

### Steps to Reproduce

[Example code](https://github.com/joshallenit/issue-robolectric-autocompletetextview/blob/main/app/src/test/java/com/example/reproduce_robo_autocomplete/AutoCompleteTextViewTest.kt)
1. Type into an AutoCompleteTextView  such that a popup should be displayed
2. Try to click on the text view

*Expected result*

- Can find and click on the auto-complete text view

*Actual result*
- Drop down item cannot be found

### Robolectric & Android Version

Robolectric 4.7.3, Android SDK 31

### Link to a public git repo demonstrating the problem:

https://github.com/joshallenit/issue-robolectric-autocompletetextview

**Comment by maanuanubhav999** [comment]
Thanks for raising this, I am also having a hard time testing it. Is there any other known methods by which we could click on the drop down selection ( with extra implementation)

**Comment by joshallenit** [comment]
> ...  Is there any other known methods by which we could click on the drop down selection ( with extra implementation)

@maanuanubhav999 Nope, because the dropdown item is never inflated and so does not exist to click on.

**Comment by utzcoz** [comment]
@joshallenit What about trying native graphics mode with latest Robolectric?
